tests/client/core/api/test_api.js

```diff
@@ -7,18 +7,26 @@ import { ErrorHandler } from 'core/errorHandler';
 import { unexpectedSuccess } from 'tests/client/helpers';
 
 
+const defaultHeaders = {
+  'Content-Type': 'application/json',
+};
+
+function generateHeaders(headerData = defaultHeaders) {
+  return {
+    get: (name) => headerData[name],
```

Comment:
Our code just polyfills it though... right? I'd hate to write code, even test code, that fails to run in Safari (rendering all our tests useless). 😢